### PR TITLE
Two resources added

### PR DESCRIPTION
**Pull request by krishamehta**
1. Django official doc
2. Sentdex video tutorial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Django REST framework is a powerful and flexible toolkit for building Web APIs.
 
 1. DRF [video tutorials playlist](https://www.youtube.com/playlist?list=PLEsfXFp6DpzTOcOVdZF-th7BS_GYGguAS) for a blog from [CodingEntrepreneurs](https://www.youtube.com/user/CodingEntrepreneurs)
 
+2. Django Web Development [Sentdex Video Tutorial](https://www.youtube.com/watch?v=FNQxxpM1yOs&list=PLQVvvaa0QuDeA05ZouE4OzDYLHY-XH-Nd)
```
Comment (AvaisP) [inline README.md]: This should be below django girls in the django section and not the DRF section

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ List of resources to get started with software development.
 ### Django
 
 Django is server-side python framework that allows developers to create web apps with little hassle. It is a high level framework and is hence even suitable for beginners. 
-
+[Official Documentation](https://docs.djangoproject.com/en/1.11/)
```
Comment (AvaisP) [inline README.md]: Create a different documentation section within Django and add this

**Comment by krishamehta** [comment]
Done

**Comment by AvaisP** [comment]
@krishamehta the second thing is still left

**Pull request merged**